content/techniques/validation.md

```diff
@@ -24,7 +24,7 @@ $ npm i --save class-validator class-transformer
 
 > info **Hint** The `ValidationPipe` is exported from the `@nestjs/common` package.
 
-Because this pipe uses the `class-validator` and `class-transformer` libraries, there are many options available. You configure these settings via a configuration object passed to the pipe. Following are the built-in options:
+Because this pipe uses the [`class-validator`](https://github.com/typestack/class-validator) and [`class-transformer`](https://github.com/typestack/class-transformer) libraries, there are many options available. You configure these settings via a configuration object passed to the pipe. Following are the built-in options:
```

Comment:
I believe repeating links on the the same page (but at another context) is redundant and makes the docs "hard" to follow _(as of now the user has a bunch of resources to look into)_. 

Do you think is better to drop the link for `class-validator` on L11 of this file?

Reply:
there is already a `@nestjs/class-validator` and `@nestjs/class-transformer` package here at GitHub. Maybe @kamilmysliwiec can tell us, if these packages will replace the currently used ones?

In this case, the docs should reflect these changes!